src/CISDSCResourceGeneration/functions/private/Get-ApplicableLevels.ps1

```diff
@@ -0,0 +1,20 @@
+function Get-ApplicableLevels {
```

Comment:
An .EXAMPLE here (and on the other functions) with example output will be helpful later if we decide to make any substantial changes to the template document.

Reply:
Good note I forgot to do that.